Dockerfile.dapper

```diff
@@ -9,18 +9,11 @@ RUN yum-config-manager --enable ol7_optional_latest && \
     yum -y install bash git docker vim less file curl wget && \
     # install golang 1.17
     yum install -y oracle-golang-release-el7 && \
```

Comment:
is this sill needed?

Reply:
Unfortunately yes, unless I want to go through the effort of getting rid of dapper.  What really ends up happening is dapper does the build first, then I rebuild using golang 1.20.10 in the Dockerfile and copy those binaries.